logger_test.go

```diff
@@ -51,21 +51,21 @@ func TestLogger(t *testing.T) {
 		assert.Equal(t, "[INFO]  test: this is test: who=programmer why=testing\n", rest)
 	})
 
-	t.Run("quotes values with spaces", func(t *testing.T) {
+	t.Run("drew testing formats log entries", func(t *testing.T) {
```

Comment:
Probably a testing value that should be changed?

Reply:
😳️ woops, fixed